pages/governance/proposing_a_new_market.md

```diff
@@ -89,9 +89,14 @@ Not recommended:
 - BinanceUS
 - Crypto.com
 
-Exchanges not included in the above list are not currently supported by the software. 
+Exchanges not included in the above list are not currently supported by the software.
 
-`exchange_config_json`
+## Example
+
+### Example of markets with robust oracle sources
+Here is a [list](https://docs.google.com/spreadsheets/d/1zjkV9R7R_7KMItuzqzvKGwefSBRfE-ZNAx1LH55OcqY/edit#gid=1489690476) of a few markets and their oracle sources evaluated based on the methodology above. 
```

Comment:
Should we drop the `edit#gid=xxxxx`?

Reply:
thanks! only me + legal have edit access but updated just in case.